src/app/style/module/ellipsis-inline/ellipsis-inline.scss

```diff
@@ -2,4 +2,5 @@
   overflow: hidden;
   text-overflow: ellipsis;
   white-space: nowrap;
+  font-size: 18px;
```

Comment:
`ellipsis-inline` si occupa di far apparire un ellipsis alla fine del testo qualora vada in overflow. Perche hai aggiunto questa proprieta a tale classe?

Sembra una proprietà che ha nulla a che fare con l ellipsis